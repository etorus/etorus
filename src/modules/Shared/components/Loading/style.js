import { StyleSheet, Image } from 'react-native';

export default StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'space-between',
    alignItems: 'center',
  },

  imageContainer: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'center',
  },

  brand: {
    height: 145,
    width: 245,
    resizeMode: Image.resizeMode.contain
  },

  version: {
    paddingBottom: 20,
    fontSize: 10,
    fontFamily: 'Branding',
    fontWeight: '400',
  },
})
