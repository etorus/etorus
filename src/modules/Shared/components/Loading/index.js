import React from 'react'
import style from './style'
import completeLogo from '../../../../assets/images/complete_logo.png'

import {
  View,
  Text,
  Image,
} from 'react-native'

const Loading = () =>
  <View style={style.container}>
    <View style={style.imageContainer}>
      <Image source={completeLogo} style={style.brand} />
    </View>
    <Text style={style.version}>
      Version 1.0.0
    </Text>
  </View>

export default Loading
