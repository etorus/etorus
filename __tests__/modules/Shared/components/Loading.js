import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer';

import Loading from 'modules/Shared/components/Loading'

describe('', () => {
  it('renders a Loading correctly', () => {
    expect(renderer.create(<Loading />)).toMatchSnapshot();
  });
})
