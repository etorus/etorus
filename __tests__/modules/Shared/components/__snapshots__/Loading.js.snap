// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Loading correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <Image
      resizeMode="contain"
      source={
        Object {
          "testUri": "../../../src/assets/images/complete_logo.png",
        }
      }
      style={
        Object {
          "height": 145,
          "width": 245,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "fontFamily": "Branding",
        "fontSize": 10,
        "fontWeight": "400",
        "paddingBottom": 20,
      }
    }
  >
    Version 1.0.0
  </Text>
</View>
`;
